src/lib/openjp2/j2k.c

```diff
@@ -12017,34 +12003,25 @@ static OPJ_BOOL opj_j2k_init_info(opj_j2k_t *p_j2k,
 
     /* TODO mergeV2: check this part which use cstr_info */
     /*l_cstr_info = p_j2k->cstr_info;
-
     if (l_cstr_info)  {
             OPJ_UINT32 compno;
             l_cstr_info->tile = (opj_tile_info_t *) opj_malloc(p_j2k->m_cp.tw * p_j2k->m_cp.th * sizeof(opj_tile_info_t));
-
             l_cstr_info->image_w = p_j2k->m_image->x1 - p_j2k->m_image->x0;
             l_cstr_info->image_h = p_j2k->m_image->y1 - p_j2k->m_image->y0;
-
             l_cstr_info->prog = (&p_j2k->m_cp.tcps[0])->prg;
-
             l_cstr_info->tw = p_j2k->m_cp.tw;
             l_cstr_info->th = p_j2k->m_cp.th;
-
             l_cstr_info->tile_x = p_j2k->m_cp.tdx;*/        /* new version parser */
     /*l_cstr_info->tile_y = p_j2k->m_cp.tdy;*/      /* new version parser */
     /*l_cstr_info->tile_Ox = p_j2k->m_cp.tx0;*/     /* new version parser */
     /*l_cstr_info->tile_Oy = p_j2k->m_cp.ty0;*/     /* new version parser */
 
     /*l_cstr_info->numcomps = p_j2k->m_image->numcomps;
-
     l_cstr_info->numlayers = (&p_j2k->m_cp.tcps[0])->numlayers;
-
     l_cstr_info->numdecompos = (OPJ_INT32*) opj_malloc(p_j2k->m_image->numcomps * sizeof(OPJ_INT32));
-
     for (compno=0; compno < p_j2k->m_image->numcomps; compno++) {
             l_cstr_info->numdecompos[compno] = (&p_j2k->m_cp.tcps[0])->tccps->numresolutions - 1;
     }
-
```

Comment:
Seems to be no change in this file other than removing lines. It's best to keep these type of changes minimal as it will may make future merging with the original master difficult.

Reply:
OK. Just my OCD about blank line in code.